src/components/ReadMore.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+interface ReadMoreProps {
+    children: string;
+}
+ 
+const ReadMore = ({ children } : ReadMoreProps) => {
+    const text = children;
+    const [isReadMore, setIsReadMore] = useState(true);
+    const toggleReadMore = () => {
+        setIsReadMore(!isReadMore);
+    };
+    return (
+        <p className="text">
```

Comment:
I'm not sure if ` text ` is a custom style or not (cannot see it being styled anywhere). If it's a type, just remove

Reply:
Please have a look again